### PR TITLE
Jabber: fix repeater leak with error on connecting

### LINKED ISSUE #375: Endless blood streams
Horta does not stop after it enters blood-gore-destroy mode: http://0xd34df00d.me/logs/chat/codingteam@conference.jabber.ru/2016/04/06.html#09:21:45

Need to check logs and the corresponding code.

### PR DESCRIPTION
**Pull request by ForNeVeR**
Fixes #375 with the fix proposed in [the issue comment](https://github.com/codingteam/horta-hell/issues/375#issuecomment-207769770).

### CODE REVIEW

File: src/main/scala/ru/org/codingteam/horta/protocol/jabber/JabberProtocol.scala [hunk 1]
```diff
@@ -59,30 +60,33 @@ class JabberProtocol() extends Actor with ActorLogging {
       val actor = context.actorOf(
         Props(new MucMessageHandler(locale, self, jid, nickname)), jid)
 
-      try {
+      val muc = Try({
         // TODO: Move this code to the room actor and use "let it fall" strategy ~ F
```
Comment (rexim) [inline src/main/scala/ru/org/codingteam/horta/protocol/jabber/JabberProtocol.scala]: Do we have an issue for this TODO btw?

Comment (ForNeVeR) [inline src/main/scala/ru/org/codingteam/horta/protocol/jabber/JabberProtocol.scala]: Don't think so. I'll create it now.

**Comment by rexim** [comment]
I approve the shit! Thank you!

File: src/main/scala/ru/org/codingteam/horta/protocol/jabber/JabberProtocol.scala (hunk 1 above)
Comment (rexim) [inline src/main/scala/ru/org/codingteam/horta/protocol/jabber/JabberProtocol.scala]: Thanks!

**Comment by ForNeVeR** [comment]
Thanks for the review.

**Pull request closed without merging**